package.json

```diff
@@ -31,6 +31,7 @@
     "redis": "^4.5.1",
     "string_decoder": "^1.3.0",
     "uWebSockets.js": "github:uNetworking/uWebSockets.js#v20.10.0",
+    "web-push": "^3.5.0",
```

Comment:
please use this lib that I suggest in a ticket https://github.com/appfeel/node-pushnotifications


it has 2 benefits over the one you choose:

- it's actively supported (web-push was updated last time a year ago)
- it provides support for iOS/Android which we will need in future